recipes/r-bigsnpr/meta.yaml

```
@@ -76,19 +86,19 @@ requirements:
     - r-foreach
     - r-ggplot2
     - r-magrittr
+    - r-quadprog
     - r-rmio
+    - r-rmutil 
     - r-roptim >=0.1.6
+    - r-rsqlite
     - r-runonce >=0.2.3
     - r-vctrs
+    - r-xgboost
```

Move to `test` only.

```suggestion
    - r-rmio
    - r-roptim >=0.1.6
    - r-runonce >=0.2.3
    - r-vctrs
```